tests/lockless_queue/enc/optee/user_ta_header_defines.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* Licensed under the MIT License. */
+#pragma once
+
+#define TA_UUID                                            \
+    { /* eb99d409-3d52-439c-b374-87f664136434 */           \
+        0xeb99d409, 0x3d52, 0x439c,                        \
+        {                                                  \
+            0xb3, 0x74, 0x87, 0xf6, 0x64, 0x13, 0x64, 0x34 \
+        }                                                  \
+    }
+
+#define TA_FLAGS (TA_FLAG_EXEC_DDR)
+#define TA_STACK_SIZE (12 * 1024)      /* 12 KB */
```

Comment:
It seems like these should be generated from the enclave properties (oeinfo).

Reply:
Idem re. `OE_SET_ENCLAVE_OPTEE`.